components/dialog/dialog-styles.js

```diff
@@ -94,13 +94,27 @@ export const dialogStyles = css`
 		height: 100%;
 	}
 
+	:host([critical]) .d2l-dialog-highlight {
+		background-color: var(--d2l-color-cinnabar);
+		border-radius: 0.4rem 0.4rem 0 0;
+		height: 0.4rem;
+		margin-inline-start: -1px;
+		margin-top: -1px;
+		position: absolute;
+		top: 0;
+		width: calc(100% + 1px);
```

Comment:
Is this a little less awkward?
```suggestion
		position: absolute;
		inset: -1px;
```

Reply:
Definitely!